tests/src/test/scala/com/nvidia/spark/rapids/GpuExpressionTestSuite.scala

```diff
@@ -175,8 +175,7 @@ abstract class GpuExpressionTestSuite extends SparkQueryCompareTestSuite {
   }
 
   def compareStringifiedDecimalsInSemantic(expected: String, actual: String): Boolean = {
```

Comment:
nit: Should we remove this method entirely? This was added to allow us to compare Decimals in a special way, but if we match it 100% now, we can compare them properly with no issues.

It is a bit of a thread to pull on though.

Reply:
I tried to do that, but I had to have a `comparisonFunc` and I don't know enough about how to remove that requirement.

Reply:
Thanks for the help, @revans2. This is fixed now and this method is gone.